cmd/vsphere/main.go

```diff
@@ -50,9 +51,14 @@ func main() {
 	// Initialize machine actuator.
 	machineActuator := machine.NewActuator(machine.ActuatorParams{
 		Client:        mgr.GetClient(),
+		RestConfig:    mgr.GetConfig(),
```

Comment:
I think we could pass `mgr.GetAPIReader()` here and rely on it further down to fetch objects beyond the cache client namespace

Reply:
https://kubernetes.slack.com/archives/CAR30FCJZ/p1580118306075000

Reply:
Fixed.